code/modules/hivemind/mobs.dm

```diff
@@ -326,9 +326,82 @@
 //Higher speed than normal
 //Slighly higher speaking chance
 //Appears from hive spawner and Mechiver
-//This is simply preparing room for a new hivemob
+//Appears rarely than bomber or stinger
 //////////////////////////////////////////////////////////////////////////////
 
+/mob/living/simple_animal/hostile/hivemind/lobber
+	name = "Lobber"
+	desc = "A little cleaning robot, filled with a unknown liquid. He looks happy, a bit too happy."
```

Comment:
```suggestion
	desc = "It's a little cleaning robot. This one appears to have been modified to fling acid intended for janitorial uses. It also appears to have it's targeting protocols overridden..."
```

Reply:
does "It's a little cleaning robot. This one appears to have it's cleaning solutions replaced with a weird goo. It also appears to have it's targeting protocols overridden..." sound good?

Reply:
'acidic goo', otherwise yeah

Reply:
"goo" or "electrolyzed goo". I would like to keep it consistent with the projectile name since that (the projectile name) is visible to the players